index.bs

```diff
@@ -346,6 +346,11 @@ A <dfn interface>DatagramDuplexStream</dfn> is a generic duplex stream.
 interface DatagramDuplexStream {
   readonly attribute ReadableStream readable;
   readonly attribute WritableStream writable;
+
+  undefined setIncomingExpirationDuration(double ms);
+  undefined setOutgoingExpirationDuration(double ms);
+  undefined setIncomingHighWaterMark(long highWaterMark);
+  undefined setOutgoingHighWaterMark(long highWaterMark);
```

Comment:
When it comes to highwaterMark, there's potential confusion with the [same-named concept in the streams spec](https://streams.spec.whatwg.org/#cqs-class):
```js
const stream = new WritableStream(
  { ... },
  new CountQueuingStrategy({ highWaterMark: 5 })
);
```
But I'm not 100% sure it's the same thing. That API I think is generally understood to be the buffer in the ReadableStream/WritableStream itself, distinct from its underlying source, whereas in our case, we have a setup at least on the writable side where we pre-fill the underlying sink's write-queue (for throughput and threading purposes), and what we really mean to control here is the depth of that write queue. But maybe this is a detail and that from the viewpoint of a producer (or consumer in the readable case) this is close enough to the the same thing?

Reply:
They represent the same concept, the length of the queue in a stream. We have them in the underlying source and sink because removing stale chunks is not supported in general streams. Given that I think it's good to keep the same name. Web developers cannot control the buffer lengths in the streams, but instead they can control these values.